common/djangoapps/student/views.py

```diff
@@ -567,9 +567,17 @@ def dashboard(request):
 
     message = ""
     if not user.is_active:
+        email_provider = user.email.split("@")[1].split(".")[0].lower()
```

Comment:
This a dangerous approach, give how much domains can vary. I'd suggest the following:
 
 - First, put it in a function so we can test it.
 - Second, use a safer check that is guaranteed to not throw an exception all the time such as
   - `user.email.endswith('gmail.com')`
   - or, `'gmail.' in user.email`, this works better for `Yahoo.fr` and `hotmail.co.uk`

Other than this, it's cool 😄.